cfme/tests/containers/test_containers_projects_summary.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from itertools import product
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.project import Project
+from cfme.web_ui import CheckboxTable
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+PROJECTS_PROPERTIES_FIELDS = ['Name', 'Creation timestamp', 'Resource version']
+PROJECTS_RELATIONSHIPS_FIELDS = ['Containers Provider', 'Routes', 'Services', 'Replicators',
+                                 'Pods', 'Nodes']
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
```

Comment:
we must start using ids. 
if there is no id for this object please ask devs to add one